Add Chinese translation of "js_alerts_prompts_and_confirmations"

alaahong opened the pull request:
### Description

Add Chinese translation of "js_alerts_prompts_and_confirmations"

- webdriver/js_alerts_prompts_and_confirmations.zh-cn.md


### Motivation and Context
### Types of changes
- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [x] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist
- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

Harsha509 commented:
Hi @alaahong,

Thanks for the PR, it looks good. I have one point here, you've marked "**Added new translation (and I also added a notice to each document missing translation)**" in types of changes.

It should be '**Improved translation**' i think.

Regards,
Harsha

alaahong commented:
> Hi @alaahong,
> 
> Thanks for the PR, it looks good. I have one point here, you've marked "**Added new translation (and I also added a notice to each document missing translation)**" in types of changes.
> 
> It should be '**Improved translation**' i think.
> 
> Regards,
> Harsha

Thanks, changed to  '**Improved translation**'  now.
